mandatory/2-function-creation.js

```diff
@@ -25,7 +49,22 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  let array = arrayOfStrings;
+  for (let i = 0; i < array.length; i++) {
+    array[i] = array[i].trim();
+    array[i] = array[i].toLowerCase();
+
+    if (array[i].includes('/')) {
+
+      index = array[i].indexOf('/');
+      array[i] = array[i].slice(index + 1, array[i].length);
+
+    }
```

Comment:
There is a simpler way of doing this, with the String replace() method.

Reply:
thanks, Karen, just used replace now as replace('/',''). replace is so simple.